tools/cldr-apps/src/main/webapp/js/redesign.js

```diff
@@ -8,7 +8,11 @@
 /*
  * Startup function
  */
-$(function() {
+require(["dojo/ready"], function(ready) {
+	const priority = 2000; // after cldrGui.run (1), before cldrReviewStartup (2001); cf. default Dojo (1000)
+	ready(priority, cldrRedesignStartup);
```

Comment:
It doesn't actually seem to matter whether priority is more or less than 1000 here, as long as it's more than 1 -- but there are other problems that may involve priority (order in which dojo modules are loaded) -- console log:
dojo/parser::parse() error SyntaxError: invalid regular expression flag r
    Dojo 61
    loadStui survey.js:2786
    showV CldrSurveyVettingLoader.js:76
    Dojo 36
parser.js.uncompressed.js:900:5

Error parsing in _ContentSetter#Setter_DIV_0 SyntaxError: invalid regular expression flag r
    Dojo 61
    loadStui survey.js:2786
    showV CldrSurveyVettingLoader.js:76
    Dojo 36
ContentPane.js.uncompressed.js:598:4

Error dijit_Dialog_0 running custom onLoad code: This deferred has already been resolved ContentPane.js.uncompressed.js:443:4
MyLoad: /cldr-apps/SurveyAjax?what=menus&_=root&locmap=true&s=47784E967A10563D19BD97AC0937D079&cacheKill=1607885253499 for initial menus for root CldrSurveyVettingLoader.js:467:14